<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>184</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDAnggotaLabel">
      <property name="text">
       <string>ID Keuangan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editID"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaAnggotaLabel">
      <property name="text">
       <string>Tanggal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editTanggal"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editKeterangan"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Keterangan Transaksi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telpLabel">
      <property name="text">
       <string>Jenis Transaksi</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Nominal (angka)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editJumlah"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboJenis">
      <item>
       <property name="text">
        <string>Pemasukan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pengeluaran</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tabelKeuangan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>711</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Keuangan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Keterangan Transaksi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jenis Transaksi</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nominal (angka)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>571</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
